control/plugin/plugin.go

```diff
@@ -21,21 +21,10 @@ package plugin
 
 // WARNING! Do not import "fmt" and print from a plugin to stdout!
 import (
-	"bytes"
-	"crypto/rsa"
-	"encoding/json"
-	"fmt"
-	"io" // Don't use "fmt.Print*"
-	"net"
-	"net/http"
-	"net/rpc"
-	"net/rpc/jsonrpc"
-	"regexp"
-	"runtime"
+	"crypto/rsa" // Don't use "fmt.Print*"
```

Comment:
This comment `// Don't use "fmt.Print*"` should be moved up below the warning or removed.